pkgs/development/python-modules/phonemizer/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, substituteAll
+, buildPythonApplication
+, fetchPypi
+, python3Packages
+, pkgs
+, joblib
+, segments
+, attrs
+, espeak-ng
+, festival
+, pytestCheckHook
+, pytestrunner
+, pytestcov
+}:
+
+buildPythonApplication rec {
+  pname = "phonemizer";
+  version = "2.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "127n4f10zxq60qd8xvlc1amji4wbghqb90rfp25rzdk716kvgwab";
+  };
+
+  postPatch = ''
+    sed -i -e '/\'pytest-runner\'/d setup.py
+  '';
+
+  patches = [
+    (substituteAll {
+      src = ./backend-paths.patch;
+      espeak = "${lib.getBin espeak-ng}/bin/espeak";
+      festival = "${lib.getBin festival}/bin/festival";
```

Comment:
Should we just not include festival into nixpkgs than? It is not really maintained anymore.

Reply:
Fine by me.